package.json

```diff
@@ -77,7 +77,7 @@
   },
   "dependencies": {
     "@artsy/backbone-mixins": "2.0.0",
-    "@artsy/cohesion": "2.22.0",
+    "@artsy/cohesion": "4.4.0",
```

Comment:
@dzucconi - did you ever learn why force hadn't been updated with the latest major versions?

Reply:
https://github.com/artsy/force/pull/8454 was failing due to some unrelated CI thing (something about the image) — it's green now so I just merged.

Reply:
I'm pretty sure major versions don't automatically merge though.

Reply:
oh cool, yeah didn't know if there was some helpers RFC work in progress blocking that from being merged 👍